website/docs/docs/core/connect-data-platform/connection-profiles.md

```diff
@@ -177,10 +177,23 @@ There are multiple ways to direct dbt to a different location for your `profiles
 ### 1. Use the `--profiles-dir` option when executing a dbt command
 This option can be used as follows:
 
+<VersionBlock lastVersion="1.4">
+
+ ```
+$ dbt --profiles-dir path/to/directory run
+ ```
+
+ </VersionBlock>
+
+
+<VersionBlock firstVersion="1.5">
+
  ```
 $ dbt run --profiles-dir path/to/directory
  ```
 
+ </VersionBlock>
+
```

Comment:
Could we remove all the versioning here? I think the following should work for all version ranges (including 1.4 and 1.5):

```
$ dbt run --profiles-dir path/to/directory
```

Reply:
you had pointed the file out earlier so i took it to mean that i should update it. sorry for the misunderstanding!